gems/pending/metadata/linux/LinuxUtils.rb

```diff
@@ -141,6 +141,44 @@ def self.parse_systemctl_list(lines)
       end.compact!
     end
 
+    def self.collect_interface(interfaces, interface)
```

Comment:
@blomquisg added tests, you can see that eth0 and br-ex are sharing mac addr, that shows which interface is connected to bridge